aea/configurations/schemas/aea-config_schema.json

```diff
@@ -110,6 +110,12 @@
     },
     "decision_maker_handler": {
       "type": "object"
+    },
+    "skill_exception_policy": {
+      "$ref": "definitions.json#/definitions/skill_exception_policy"
+    },
+    "default_routing": {
+      "type": "object"
```

Comment:
This could do with more typing/constraints..not my area.. @marcofavorito ?

Reply:
yes. We could use:

- `patternProperties` to validate the keys (they must be public ids)
- the usual `$ref` to the public id definition.

For the first bit: I can't see an elegant way. We have to copy the regular expression in `definitions.json#public_id` and put it there.

You can see an example of `patternProperties` in `aea-config_schema.json` when we define `ledger_apis`.

Reply:
@marcofavorito check out what I did, hope this is correct